lib/debugger.js

```diff
@@ -8,12 +8,13 @@ var Net = require('net'),
 /**
 * @param {Number} port
 */
-function Debugger(port){
+function Debugger(host, port){
```

Comment:
Ditto, can we change the argument order to `(port, host)` for backwards compatibility?